mysite/blog/templates/blog/post/detail.html

```diff
@@ -4,11 +4,44 @@
 {% endblock %}
 {% block content %}
     <h1>{{ post.title }}</h1>
+    {% if messages %}
+    <ul class="messages">
+        {% for message in messages %}
+        <li {% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+        {% endfor %}
+    </ul>
+    {% endif %}
     <p class="date">Published {{ post.publish }} by {{ post.author }}</p>
     {{ post.body|linebreaks }}
     <p>
         <a href="{% url 'blog:post_share' post.id %}">
         Share this post
         </a>
     </p>
-{% endblock %}
+    {% with post.get_active_comments.count as total_comments %}
+    <h2>
+      {{ total_comments }} comment{{ total_comments|pluralize }}
+    </h2>
+  {% endwith %}
+  {% for comment in post.get_active_comments %}
+    <div class="comment">
+      <p class="info">
+        Comment {{ forloop.counter }} by {{ comment.name }}
+        {{ comment.created }}
+      </p>
+      {{ comment.body|linebreaks }}
+    </div>
+  {% empty %}
+    <p>There are no comments yet.</p>
+  {% endfor %}
+  {% if new_comment %}
+    <h2>Your comment has been added.</h2>
+  {% else %}
+    <h2>Add a new comment</h2>
+    <form method="post">

```

Comment:
```suggestion
    <form method="post" action="url of <int:post_id>/comment/">

```
Check the bellow comment on handling this in a separate view